packages/store/addon/-private/ts-interfaces/ds-model.ts

```diff
@@ -1,11 +1,10 @@
-import { Record } from './record';
 import RSVP from 'rsvp';
 import EmberObject from '@ember/object';
 import { JsonApiValidationError } from './record-data-json-api';
 import { RelationshipSchema } from './record-data-schemas';
 
 // Placeholder until model.js is typed
-export interface DSModel extends Record, EmberObject {
+export interface DSModel extends EmberObject {
```

Comment:
Missing RecordInstance here

Reply:
it's intentional, it should not be extending `RecordInstance`